realm/realm-jni/build.gradle

```diff
@@ -268,6 +316,7 @@ task clean(type: Delete) {
 
     delete fileTree(dir: "${projectDir}/../realm-library/src/main/jniLibs/", include: '**/librealm-jni*.so')
     delete fileTree(dir: "${projectDir}/../build/output/jniLibs-unstripped/", include: '**/librealm-jni*.so')
+    delete fileTree(dir: "${projectDir}/src/", include: '**/librealm-jni*-stripped.so')
```

Comment:
this so is not created by Makefile (not directly) and `make clean` is called below.